selfdrive/car/gm/interface.py

```diff
@@ -36,12 +36,27 @@ def get_steer_feedforward_acadia(desired_angle, v_ego):
     desired_angle *= 0.09760208
     sigmoid = desired_angle / (1 + fabs(desired_angle))
     return 0.04689655 * sigmoid * (v_ego + 10.028217)
+  
+  @staticmethod
+  def get_steer_feedforward_lacrosse(angle, speed):
+    ANGLE_COEF = 0.58539783
+    ANGLE_COEF2 = 0.32765082
+    ANGLE_OFFSET = 0.00460531
+    SPEED_OFFSET = 13.23075991
+    SIGMOID_COEF_RIGHT = 0.13719471
+    SIGMOID_COEF_LEFT = 0.13309956
+    SPEED_COEF = 0.06147823
+    x = ANGLE_COEF * (angle + ANGLE_OFFSET)
+    sigmoid = x / (1. + fabs(x))
+    return ((SIGMOID_COEF_RIGHT if (angle + ANGLE_OFFSET) > 0. else SIGMOID_COEF_LEFT) * sigmoid) * ((speed + SPEED_OFFSET) * SPEED_COEF) * ((fabs(angle + ANGLE_OFFSET) ** fabs(ANGLE_COEF2)))
```

Comment:
My new C3 won’t arrive until tomorrow and I’ll be on a business trip until Friday, so next weekend will be the earliest that I can attempt a test drive. Hopefully the radar faults will be gone with the new hardware. 

Regarding the steering, I would agree with you in principle but in reality the stock tuning is borderline unusable. Below 40 mph it will oscillate dangerously in corners, and even on the highway it will touch the lines on both sides of the lane. You can see this reflected in the StDev measurements noted in the tune comments. So I would prefer to leave the custom tuning in, but would like your perspective on whether it’s better to include it now or in a separate PR.

Reply:
@nuwandavek do you have any opinion on merging temporary fixes for GM if it's this bad? Or should we wait for a general solution?

@koch-cf yes, let's make it a new PR since this sounds like it might block merging car support.

Reply:
Ran a test route with the custom lateral tuning and it is 100% better.  Car is now stable as low as 25mph, whereas with the stock tuning the car would literally weave back and forth across the entire road at that speed.  Did not get a chance yet to test high-speed instability (oscillating in sweeping turns at 65mph) but I predict that's also resolved.  If the custom tuning isn't acceptable for porting the car then something else is needed or it's going to be hazardous to drive.